js/carrousel-homepage.js

```diff
@@ -0,0 +1,67 @@
+  document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Do you have source for it ? or its a plain code you have written ? if there is a source : You should mention it in top of the page !